app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+
+    command, source_file, new_file = parts[0], parts[1], parts[2]
```

Comment:
The variable `command` is being reassigned here, which can lead to confusion. Consider renaming the reassigned variable to something like `cmd` to avoid overwriting the function parameter.